elements/urth-viz-table/urth-viz-table.html

```diff
@@ -278,6 +278,7 @@
 
                     var tableContainer = this.$.tableContainer;
                     tableContainer.innerHTML = '';
+                    tableContainer.style.height = 'auto';
```

Comment:
@lbustelo mostly looking to simplify the logic / eliminate the potential for conflicts like this. Maybe `!important` overrides would be needed to accomplish via CS.